tools/archetype/binding/src/main/resources/archetype-resources/NOTICE

```diff
@@ -0,0 +1,19 @@
+This content is produced and maintained by the Eclipse SmartHome project.
+
+* Project home: https://eclipse.org/smarthome/
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.html.
```

Comment:
https://github.com/eclipse/smarthome/blob/master/NOTICE

Reply:
?

Reply:
Copy&Paste error, I was checking the blob file as well...

I meant: https://www.eclipse.org/legal/epl-2.0/